sections/04_study.md

```diff
@@ -264,121 +264,236 @@ and [#258](https://github.com/greenelab/deep-review/issues/258).`
 *We may want to be selective about what we discuss and not list every
 application in this area.*
 
+`TODO: There's inevitably a lot of overlap been this and the transcription
```

Comment:
Let's leave this TODO in place.  Initially we can focus on merging this quickly, especially because you fixed typos and trailing spaces in other sub-sections.  If we leave this open too long, it will inevitably lead to merge conflicts.  Next, we can decide how to combine or re-organize this section and the TF section.